rust-toolchain

```diff
@@ -0,0 +1 @@
+stable
```

Comment:
I think we should not require a specific toolchain.

Reply:
this is only necerssary so that `rustup` can automatically determine the toolchain being used and automatically select one

Reply:
otherwise, i'd need to set a directory override within `rustup` or set a global toolchain

Reply:
I understand, but we do have beta builds running on the CI (not enabled right now, because of an unresolvable conflict with clippy). The build is configured with a `rustup default beta`, so how should we test the repository with beta or nightly builds, if the toolset is fixed in the repository?

Reply:
you would do a directory override in the ci, with `rustup override set <channel>`